media/css/devreg/status.styl

```diff
@@ -1,5 +1,28 @@
 @import 'lib';
 
+#disable-addon {
```

Comment:
classes are usually preferred over IDs for styling since IDs have too much specificity